### PR TITLE
Noyon assignment

### PR DESCRIPTION
**Pull request by noyonict**
I have added my assignment here. Plz check and approve

### CODE REVIEW

**Comment by noyonict** [comment]
Plz anyone review my PR

**Review by mizanur-007 — APPROVED** [review]
Looks ok

File: noyon/8_functions.py
Comment (shahriarmohim-bs23) [inline noyon/8_functions.py:1]: all ok

**Review by shahriar-mohim007 — APPROVED** [review]

**Review by mtk0019 — APPROVED** [review]
Everytning looks okay

**Pull request merged**